### PR TITLE
IDENTITY-6177: Adding tests for admin user from SCIM feature

### PR DESCRIPTION
**Pull request by emswbandara**

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim/SCIMServiceProviderUserTestCase.java [hunk 1]
```diff
@@ -62,10 +63,11 @@
...
     private SCIMClient scimClient;
+    private AutomationContext automationContext = null;
 
     @BeforeClass(alwaysRun = true)
     public void initiate() throws Exception {
-        AutomationContext automationContext = new AutomationContext("IDENTITY", TestUserMode.SUPER_TENANT_ADMIN);
+        automationContext = new AutomationContext("IDENTITY", TestUserMode.SUPER_TENANT_ADMIN);
         provider_userInfo = automationContext.getContextTenant().getContextUser();
         backendUrl = automationContext.getContextUrls().getBackEndUrl();
         scim_url = backendUrl.substring(0, 22) + "/wso2/scim/";
```
Comment (nuwandiw) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim/SCIMServiceProviderUserTestCase.java]: Extend ISIntegrationTest class and get the backend url from isServer object

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim/SCIMServiceProviderUserTestCase.java [hunk 2]
```diff
@@ -158,6 +159,36 @@ public void getUserMe() {
...
+            userInfo.setUserName(this.isServer.getSuperTenant().getTenantAdmin().getUserName());
+            userInfo.setPassword(this.isServer.getSuperTenant().getTenantAdmin().getPassword());
+            BasicAuthInfo encodedBasicAuthInfo = SCIMUtils.getBasicAuthInfo(userInfo);
+            //create resource endpoint to get admin user information.
+            Resource userResource = restClient.resource(scim_url + SCIM_USER_ENDPOINT);
+            String response = userResource.header(SCIMConstants.AUTHORIZATION_HEADER, encodedBasicAuthInfo
+                    .getAuthorizationHeader())
+                    .contentType(SCIMConstants.APPLICATION_JSON).accept(SCIMConstants.APPLICATION_JSON).get(String
+                            .class);
+            Assert.assertTrue(response.contains(
```
Comment (pulasthi7) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim/SCIMServiceProviderUserTestCase.java]: Better to static import Assert.* methods

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim/SCIMServiceProviderUserTestCase.java [hunk 3]
```diff
@@ -33,27 +33,27 @@
...
 import java.rmi.RemoteException;
 
-public class SCIMServiceProviderUserTestCase {
+public class SCIMServiceProviderUserTestCase extends ISIntegrationTest {
     private static final Log log = LogFactory.getLog(SCIMServiceProviderUserTestCase.class);
     private static final String USERNAME = "SCIMUser1";
     public static final String PASSWORD = "password1";
     public static final String ADMIN_ROLE = "admin";
     public static final String SCIM_ME_ENDPOINT = "Users/me";
+    public static final String SCIM_USER_ENDPOINT = "Users";
```
Comment (pulasthi7) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim/SCIMServiceProviderUserTestCase.java]: Do we need this to be public (and the ones above..)?

**Comment by JKAUSHALYA** [comment]
Merged with https://github.com/wso2/product-is/pull/1464

**Pull request closed without merging**